dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/ProcessUtils.java

```diff
@@ -59,7 +59,7 @@
     /**
      * Expression of PID recognition in Windows scene
      */
-    private static final Pattern WINDOWSATTERN = Pattern.compile("(\\d+)");
+    private static final Pattern WINDOWSATTERN = Pattern.compile("\\((\\d+)\\)");
```

Comment:
There is a problem with the regular expression, please check.

![image](https://user-images.githubusercontent.com/37063904/110419379-2d844f80-80d4-11eb-99d7-059b310b81e6.png)

Reply:
![image](https://user-images.githubusercontent.com/20266837/110423391-ec903900-80db-11eb-9786-ecc14ea33435.png)
test result is like this,not contains "()".and if the threads should be excluded, "\w+\((\d+)\)" take effect，just like this
![image](https://user-images.githubusercontent.com/20266837/110423653-6aecdb00-80dc-11eb-9375-20d7e0d768d2.png)
if you think the second one make sense,i'll fix it

Reply:
I think it should be the second one, but I checked the regularization, and the result is not correct

![image](https://user-images.githubusercontent.com/37063904/110458713-0d21b880-8107-11eb-9f6e-dbecef0d8ed3.png)

Reply:
matcher.group returns the match string in (), website regex utils find the whole match string,they are not the same.You can try it in java code